qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -539,20 +539,28 @@
             input_dev_type_input3 = tablet
         - gagent_ssh_public_key_injection:
             only Linux
-            guest_user = "fedora"
-            guest_user_passwd = "redhat"
-            guest_homepath = /home/${guest_user}
             gagent_check_type = ssh_public_key_injection
-            cmd_add_user_set_passwd = useradd ${guest_user} && echo ${guest_user_passwd} | passwd --stdin ${guest_user}
-            set_setenforce = "setenforce %s"
-            cmd_remove_user = userdel -rf ${guest_user}
+            set_sebool = "setsebool virt_qemu_ga_read_nonsecurity_files on ; setsebool virt_qemu_ga_manage_ssh on"
             cmd_clean_keys = rm -rf ~/.ssh/*
             ssh_keygen_cmd =  "ssh-keygen -t rsa -P "" -f ~/.ssh/id_rsa"
+            cmd_get_hostkey = "cat ~/.ssh/id_rsa.pub"
+            variants:
+                - root:
+                    guest_user = "root"
+                    guest_homepath = /${guest_user}
+                    test_login_guest = ssh ${guest_user}@%s -o stricthostkeychecking=no ls ${guest_homepath}
+                    output_check_str = 'anaconda-ks.cfg'
```

Comment:
Anywhere do we use this parameter?

Reply:
@vivianQizhu you're right, that part has been updated. please take a look